docker-compose-developer.yml

```diff
@@ -82,6 +82,22 @@ services:
     profiles:
       - engine
 
+  oncall_telegram_polling:
```

Comment:
is this new container missing the following?
```yaml
user: *oncall-user
```

Reply:
done